### PR TITLE
[SITES-512] Feature/aggregate news by distribution

### PR DESCRIPTION
**Pull request by ltankey**
This PR ensures that a `/:section/news` block includes any news item for which the section is marked as a distribution.

### CODE REVIEW

File: app/controllers/editorial/news_controller.rb [hunk 1]
```diff
@@ -35,10 +35,13 @@ def create
           # TODO: move this into a form validator
           # Prevents distribution list being created for publisher
```
Comment (tessereth) [inline app/controllers/editorial/news_controller.rb]: Is this comment now incorrect?

**Comment by tessereth** [comment]
LGTM

**Comment by micapam** [comment]
So this duplicates the `section` into `sections` collection - is that right ?

**Comment by ltankey** [comment]
Correct. Originally I had attempted to combine two scopes but was getting grief getting the query from the `Section` working, as it would have to apply an `or` statement on both published and distribution relationships.

It's far simpler to treat all sections as the same for distributions, and restrict what gets rendered where in the views.

**Comment by maxious** [comment]
LGTM but Merge conflicts

**Pull request merged**